src/txkube/test/test_authentication.py

```diff
@@ -167,7 +168,7 @@ def _authorized_request(self, token, headers,
         assertNoResult(self, d)
         [(host, port, factory, _, _)] = reactor.tcpClients
 
-        addr = HOST_MAP.get(kubernetes_host.decode("ascii"), None)
+        addr = HOST_MAP.get(ensure_unicode(kubernetes_host), None)
```

Comment:
`kubernetes_host` looks like it's always bytes to me.

Reply:
I removed this in the updated patch.